apps/sveltekit-example-app/src/app.d.ts

```diff
@@ -1,13 +1,14 @@
 // See https://kit.svelte.dev/docs/types#app
 // for information about these interfaces
-import type { Session, User } from '@packages/auth-lucia'
+
+import type { ApiClient } from '@packages/api/client'
 
 declare global {
   namespace App {
     // interface Error {}
     interface Locals {
-      session: Session | null
-      user: User | null
+      api: ApiClient['api']
```

Comment:
The hooks.server.ts will ensure a configured API Client is always available via locals context